src/test/java/jenkins/plugins/office365connector/model/MentionTest.java

```diff
@@ -0,0 +1,67 @@
+package jenkins.plugins.office365connector.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+/**
+ * Strict tests for Mention class (outer entity for Adaptive Card mentions).
+ * @author Ammar Zain (AmmarOFA@github)
+ */
+class MentionTest {
+
+    @Test
+    void defaultConstructor_FieldsAreNull() {
+        // given / when
+        Mention mention = new Mention();
+
+        // then
+        assertThat(mention.getText(), equalTo(null));
```

Comment:
this is initial value that will never be valid in runtime - does it make sense to test it?